rotkehlchen/tests/data_migrations/test_migrations.py

```diff
@@ -107,7 +107,7 @@ def test_migration_1(rotkehlchen_api_server):
         assert rotki.data.db.get_settings(cursor).last_data_migration == 1
 
 
-@pytest.mark.parametrize('use_custom_database', ['data_migration_v0.db'])
+@pytest.mark.parametrize('use_custom_database', ['data_migration_v0_original.db'])
```

Comment:
What is this change for?

Reply:
I had a similar issue as the one with the filters in the db migration v24 and solved it by having manually edited the database. Wrote a custom insert and removed the duplicated file instead since is the correct solution here